website/aws.erb

```diff
@@ -3400,6 +3400,9 @@
                                 <li>
                                     <a href="/docs/providers/aws/r/workspaces_ip_group.html">aws_workspaces_ip_group</a>
                                 </li>
+                                <li>
+                                    <a href="/docs/providers/aws/r/workspace">aws_workspace</a>
```

Comment:
Needs to be `<a href="/docs/providers/aws/r/workspace.html">` to fix the website build error.

Reply:
Fixed